src/game/object_helpers.c

```diff
@@ -185,6 +185,26 @@ Gfx *geo_switch_anim_state(s32 callContext, struct GraphNode *node) {
     return NULL;
 }
 
+s16 gRoomOverride = -1;
+
+void set_room_override(s16 room) {
+    gRoomOverride = room;
+}
+
+s16 get_room_at_pos(f32 x, f32 y, f32 z) {
```

Comment:
this function seems to only be useful in very specific situations. it would probably be more useful if you could enable `gFindFloorIncludeSurfaceIntangible` for your own ray casts, since you can already just get the room parameter from the surface you find.

Reply:
having `gFindFloorIncludeSurfaceIntangible` true does change the behavior of `find_floor` but its fine if you just remove the function too.